layouts/joomla/system/message.php

```diff
@@ -43,25 +44,14 @@
 	->useStyle('webcomponent.joomla-alert')
 	->useScript('webcomponent.joomla-alert');
 
+$output = null;
+
+if (is_array($msgList) && !empty($msgList)) :
+	foreach ($msgList as $type => $msgs) :
+		$output = LayoutHelper::render('joomla.system.joomla-alert', ['alertType' => $alert[$type], 'type' => $type, 'msg' => $msgs]);
+	endforeach;
+endif;
 ?>
 <div id="system-message-container" aria-live="polite">
-	<div id="system-message">
-		<?php if (is_array($msgList) && !empty($msgList)) : ?>
-			<?php foreach ($msgList as $type => $msgs) : ?>
-				<joomla-alert type="<?php echo $alert[$type] ?? $type; ?>" dismiss="true">
-					<?php if (!empty($msgs)) : ?>
-						<div class="alert-heading">
-							<span class="<?php echo $type; ?>"></span>
-							<span class="sr-only"><?php echo Text::_($type); ?></span>
-						</div>
-						<div class="alert-wrapper">
-							<?php foreach ($msgs as $msg) : ?>
-								<div class="alert-message"><?php echo $msg; ?></div>
-							<?php endforeach; ?>
-						</div>
-					<?php endif; ?>
-				</joomla-alert>
-			<?php endforeach; ?>
-		<?php endif; ?>
-	</div>
+	<div id="system-message"><?php echo $output; ?></div>
```

Comment:
```suggestion
	<?php if (is_array($msgList) && !empty($msgList)) : ?>
	<div id="system-message"></div>
	<?php else: ?>
	<div id="system-message">
		<?php if (is_array($msgList) && !empty($msgList)) : ?>
			<?php foreach ($msgList as $type => $msgs) : ?>
				<joomla-alert type="<?php echo $alert[$type] ?? $type; ?>" dismiss="true">
					<?php if (!empty($msgs)) : ?>
						<div class="alert-heading">
							<span class="<?php echo $type; ?>"></span>
							<span class="sr-only"><?php echo Text::_($type); ?></span>
						</div>
						<div class="alert-wrapper">
							<?php foreach ($msgs as $msg) : ?>
								<div class="alert-message"><?php echo $msg; ?></div>
							<?php endforeach; ?>
						</div>
					<?php endif; ?>
				</joomla-alert>
			<?php endforeach; ?>
		<?php endif; ?>
	</div>
	<?php endif; ?>
```

This would solve the problem too, but I'm sure there is a better way todo it

Reply:
it prevents loading another file. 
line 60 and 76 are obsolete now. Is that correct?